src/status_im/contexts/profile/contact/view.cljs

```diff
@@ -20,5 +20,8 @@
       :cover-color      (or customization-color colors/primary-50)
       :background-color (colors/theme-colors colors/white colors/neutral-95 theme)
       :page-nav-props   {:right-side [{:icon-name :i/options
-                                       :on-press  not-implemented/alert}]}}
+                                       :on-press  (fn []
+                                                    (rf/dispatch [:show-bottom-sheet
+                                                                  {:content (fn []
+                                                                              [actions/view])}]))}]}}
```

Comment:
Not sure if this should be wrapped in use-effect like the ui guideline says.

Reply:
Yeah I think callbacks need to be wrapped in a `use-callback` function, and that needs to be sorta structured like this example:
```
(defn view
  []
  (let [on-press (rn/use-callback 
                  (fn [] 
                    (rf/dispatch [:show-bottom-sheet 
                                  {:content (fn [] [actions/view])}])))]
    [rn/pressable
     {:on-press on-press}]))
```

Reply:
resolved please check again @seanstrom